core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -152,102 +144,18 @@ public void init() throws StoreInitializationException {
     }
   }
 
-  public static void addSecurityConfigsToClientProperties(SchemaRegistryConfig config,
-                                                          Properties props) {
-    addSslConfigsToClientProperties(config, props);
-    addSaslConfigsToClientProperties(config, props);
+  public static void addSchemaRegistryConfigsToClientProperties(SchemaRegistryConfig config,
+                                                                Properties props) {
+    props.putAll(config.originalProperties());
```

Comment:
Right, I got that the non-prefixed is added before the prefixed. My concern though is cases where people turn on https for SR but *don't* have any security configs for Kafka. Not sure that would be common, just noting that it would now be impossible and a change in behavior.